frontend/lib/map/map/map.dart

```diff
@@ -161,10 +161,14 @@ class _MapState extends State<Map> implements MapController {
     if (features.isNotEmpty) {
       final mapPoint = await controller.toLatLng(point);
       int distSort(a, b) {
-        final cA = a["geometry"]["coordinates"] as List<double>;
-        final cB = b["geometry"]["coordinates"] as List<double>;
-        final dA = math.sqrt(math.pow(cA[0] - mapPoint.longitude, 2) + math.pow(cA[1] - mapPoint.latitude, 2));
-        final dB = math.sqrt(math.pow(cB[0] - mapPoint.longitude, 2) + math.pow(cB[1] - mapPoint.latitude, 2));
+        final cA = a["geometry"]["coordinates"];
+        final cB = b["geometry"]["coordinates"];
+        final longitudeA = cA[0] as double;
+        final latitudeA = cA[1] as double;
+        final longitudeB = cB[0] as double;
+        final latitudeB = cB[1] as double;
+        final dA = math.sqrt(math.pow(longitudeA - mapPoint.longitude, 2) + math.pow(latitudeA - mapPoint.latitude, 2));
```

Comment:
This is just a refactor, right? This was not necessary because the API changed or something like that, right?

Reply:
Actually, I got errors and the app did crash for me because apparently `cA` and `cB` can not be cast to `List<double>`

Reply:
That is weird, are you using the same Dart and Flutter versions as the CI?

Reply:
Nearly, but it is just a runtime error.

Reply:
You should use the same versions, else stuff like that will happen over and over :)

Reply:
That is not because of this, it was just wrong :D But yes, will do